languages/csharp-dotnet-2.2/template/src/pipe.cs

```diff
@@ -0,0 +1,16 @@
+using Algorithmia;
+using __ALGO__;
+// This file takes your __ALGO__.cs file, loads the apply function and creates a Pipe wrapper around it.
+// WARNING: Do not remove this file from your project, it will make your algorithm unusable.
+
+namespace __ALGO__
+
+{
+    public class pipe
+    {
+        static int Main()
+        {
+            return AlgorithmiaPipe.Pipe.Enter(typeof(__ALGO__));
```

Comment:
maybe I need to look at the implementation of the pipe but I thought you would normally pass in a new instance of the `__ALGO__` object

Reply:
No here we actually want the type, not an object from that class